Dockerfile

```diff
@@ -123,7 +123,7 @@ ENV KOPS_TEMPLATE=/templates/kops/default.yaml
 
 # https://github.com/kubernetes/kops/blob/master/channels/stable
 # https://github.com/kubernetes/kops/blob/master/docs/images.md
-ENV KOPS_BASE_IMAGE=kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
+ENV KOPS_BASE_IMAGE=kope.io/k8s-1.10-debian-jessie-amd64-hvm-ebs-2018-08-17
```

Comment:
Use 
`kope.io/k8s-1.11-debian-stretch-amd64-hvm-ebs-2018-08-17`
as this is recommended version for k8s `1.10.11` 
Check here https://github.com/kubernetes/kops/blob/master/channels/stabl

Reply:
My fault. 
I looked the wrong line.
Sorry

Reply:
No problem @goruha , thanks!